src/core/agent/records/credentialMetadataRecord.test.ts

```diff
@@ -12,7 +12,6 @@ const mockData: CredentialMetadataRecordProps = {
   status: CredentialStatus.CONFIRMED,
   connectionId: "EEnw0sGaicPN-9gHgU62JIZOYo7cMzXjd-fpwJ1EgdK6",
   schema: "EBfdlu8R27Fbx-ehrqwImnK-8Cm79sqbAQ4MmvEAYqao",
-  identifierType: IdentifierType.Individual,
```

Comment:
@Sotatek-BaoHoanga Please re-read the acceptance criteria. This field must stay. I asked to use `Omit` on this field in the `getAcdcFromIpexGrant` only